SPECS/numpy/generate_source_tarball.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -e
+
+SRC_TARBALL="numpy-1.23.4.tar.gz"
```

Comment:
why not using $PKG_VERSION ?

Reply:
I was just using the script as given, but am cleaning up the parameters as noted.